src/BrightScriptFileUtils.ts

```diff
@@ -1,13 +1,40 @@
 export default class BrightScriptFileUtils {
     public getAlternateFileName(fileName: string): string | undefined {
-        if (fileName !== undefined && fileName.toLowerCase().endsWith('.brs')) {
-            //jump to xml file
+        if (fileName !== undefined && (fileName.toLowerCase().endsWith('.brs'))) {
             return fileName.substring(0, fileName.length - 4) + '.xml';
+        } else if (fileName !== undefined && (fileName.toLowerCase().endsWith('.bs'))) {
+            return fileName.substring(0, fileName.length - 3) + '.xml';
         } else if (fileName !== undefined && fileName.toLowerCase().endsWith('.xml')) {
-            //jump to brs file
             return fileName.substring(0, fileName.length - 4) + '.brs';
         } else {
             return undefined;
         }
     }
+    public getAlternateBrsFileName(fileName: string): string | undefined {
```

Comment:
Can you add a jsdoc comment for this method to explain its purpose and how it differs from `getAlternateFileName`?